ports/stm32/omv_spi.c

```diff
@@ -95,7 +95,7 @@ DEFINE_SPI_INSTANCE(6)
 
 static omv_spi_t *omv_spi_descr_all[6] = { NULL };
 #if defined(STM32N6)
-static DMA_NodeTypeDef OMV_ATTR_SECTION(OMV_ATTR_ALIGNED(omv_dma_nodes_all[6 * 2], 32), ".dma_buffer");
+static DMA_NodeTypeDef OMV_ATTR_SECTION(OMV_ATTR_ALIGNED(omv_dma_nodes_all[6 * 2], 32), ".d2_dma_buffer");
```

Comment:
Should this be `OMV_VOSPI_DMA_BUFFER` ?

Reply:
It could be. It's shared though for all SPI DMA. Suggestions?

Reply:
Given HPDMA isn't really needed for this, I left it with GPDMA to avoid any more changes being required.